### PR TITLE
Implement fields in Player Class

### PR DESCRIPTION
**Pull request by yevGen-S**
- setters for bicycle, guitar, computer - States
 - functions for Player work: playSong(),deliver(),realiseWebTask()
 - function for Player eat
 - functions to change 3 main stats of Player
 - functions to change States of items in Bag
 - Closes:#36

### CODE REVIEW

**Review by Bazalii — CHANGES_REQUESTED** [review]
Several things should be changed

File: Project_source/app/src/main/java/android/clicker/school_live_simulator/Classes/Main/Player.kt [hunk 1]
```diff
@@ -4,25 +4,40 @@ class Player {
     object Bag {
```
[the remark this answers is not in the document]
Comment (yevGen-S) [reply Project_source/app/src/main/java/android/clicker/school_live_simulator/Classes/Main/Player.kt]: Fixed

File: Project_source/app/src/main/java/android/clicker/school_live_simulator/Classes/Main/Player.kt [hunk 2]
```diff
@@ -52,58 +67,74 @@ class Player {
     }
 
     fun playSong(song: Song) {
-        TODO("Not yet implemented")
+        if(this.current_courses.guitar_course.isAvailable(song)){
+            addMoney(song)
```
[the remark this answers is not in the document]
Comment (yevGen-S) [reply Project_source/app/src/main/java/android/clicker/school_live_simulator/Classes/Main/Player.kt]: Fixed

File: Project_source/app/src/main/java/android/clicker/school_live_simulator/Classes/Main/Player.kt [hunk 3]
```diff
@@ -119,19 +150,19 @@ class Player {
...
     fun buyNewComputer() {
-        TODO("Not yet implemented")
+        this.items.computer.changeState(Player.Bag)
     }
     fun buyNextComputerCourse() {
-        TODO("Not yet implemented")
+        this.current_courses.computer_course.buyNextCourse(Player.Courses)
     }
 
 }
```
[the remark this answers is not in the document]
Comment (yevGen-S) [reply Project_source/app/src/main/java/android/clicker/school_live_simulator/Classes/Main/Player.kt]: Fixed

File: Project_source/app/src/main/java/android/clicker/school_live_simulator/Classes/Main/Player.kt [hunk 4]
```diff
@@ -4,25 +4,40 @@ class Player {
...
             }
     }
     object Courses {
         var guitar_course: GuitarCourseState = NullGuitarCourseState()
             set(value) {
-                TODO("Not yet implemented")
+                if(money >= value.price){
+                    reduceMoney(value.price)
+                    field = value
+                }
```
[the remark this answers is not in the document]
Comment (yevGen-S) [reply Project_source/app/src/main/java/android/clicker/school_live_simulator/Classes/Main/Player.kt]: Fixed

**Review by Bazalii — APPROVED** [review]

**Review by Endray952 — APPROVED** [review]

**Pull request merged**